### ISSUE TITLE
Add more groups that we know about

### ISSUE THREAD

**Issue by ian-codes** [ian-codes/connectbern#3]

**Comment by ian-codes** [comment]
@chagai95 Do you happen to have a list of some more groups that haven't been added yet somewhere?
Perhaps you could list some as a comment in this issue, or add them to the code yourself at your convenience.

**Comment by chagai95** [comment]
ECB https://www.ec-bern.ch/

**Comment by chagai95** [comment]
all the groups in Telegram sammlung

**Comment by ian-codes** [comment]
> all the groups in Telegram sammlung

I thought the telegram sammlung group was a collection of only UNKO groups? That's why the group was renamed in [this issue](https://github.com/ian-codes/connectbern/issues/2#issue-2267684073).

Do you mean we should have all of them individually listed as groups instead of contained in this collection?

**Comment by chagai95** [comment]
Yeah it'll be better to list then so people can find them here as well and also have a better explanation for each

**Comment by ian-codes** [comment]
Could you please put together an updated list of individual groups that we could add that haven't been added yet?

**Comment by chagai95** [comment]
yes sure! 

These are probably up to date:

<img width="984" height="711" alt="Image" src="https://github.com/user-attachments/assets/6cbd58be-4841-4cf1-b877-bf65026a2526" />

And these needs to be checked if they are up to date but most of them are:

Bern

Unko-Märkte:
💎 Bern - Hauptgruppe (https://t.me/+j6oiWwlyYiw5MjM0)
💠 Bümpliz und Bethlehem (https://tiny.cc/unkomarktbernbuempliz) 
💠 Fischermätteli, Mattenhof, Sulgenau und Weissenbühl (https://t.me/joinchat/V1agD1T8E7ZiOWY0)
💠 Zollikofen (https://tiny.cc/zusammenzollikofentg)
💠 Konolfingen (https://t.me/+0d0e2-SaejQxZTc0)
💠 Worb (https://t.me/+qRhXsX8PJws1MDk0)
💠 Gürbetal (https://t.me/+OYMZhEs90tQ1NjU0)
💠 Region Gantrisch (https://t.me/+JXMdN-Go9UY2NDM0)
💠 Bern Wittigkofen-Quartier (http://tiny.cc/wittiunko)

🧒 Kindersachen (https://t.me/BEKindersachen)
🥗 Foodsharing (https://t.me/foodsharingbe)
👕 Kleidertausch (https://t.me/kleidertauschbern)
🌱 Pflanzentausch (https://t.me/pflanzentauschen1)
🎷 Musik-Instrumente (https://t.me/MusikInstrumente_Tausch_Bern)
🎸 Musiker*innen/Bands (https://t.me/musikernetzbern)

Events:
📅 Hauptgruppe (https://t.me/bernunkoagenda)
📅 Chatgruppe (https://t.me/+UOrn5kOXNfeKYWHX) (Veranstalter*innen)
📅 Events in und um Bern (https://t.me/eventsbern)

Gesellschaftlich:
🥰 Unko-Care (https://t.me/+Hr1XrvM2JZplZmZk)
💗 Sippe von Bern (https://t.me/menschbern)
👥 Allerlei Menschliches (https://t.me/+kAkkzvPqKidlYWU0)
🔗 Connect Bern (https://t.me/connectbern)
💜 Kritische Männlichkeiten (https://t.me/+2lFNnfMSru4wMWY0)
🥦 Vegan Bern & Agglo (https://t.me/bernvegan)
🃏 Spieleabendgruppe (https://linktr.ee/gamesbern)

😌 Tipps für Therapie, Beratung, Yoga (https://t.me/joinchat/fQAk9XDEkiM4NTJk)
🏘 Wohnen - WG Suche (https://t.me/joinchat/LIezUBJgr4IRDyv8jYTWkg)
🚕 Mitfahrgelegenheit (https://tiny.cc/mitfahrenbern)
➡️ Vorwärts links (https://t.me/joinchat/E2kynkdsZXd1x7lXB6wdJw)
🏴‍☠️️ (https://t.me/ChagaiAdmin) Watch BE (https://t.me/ChagaiAdmin) (fragen)
👩‍💻 IT enthusiasts (https://t.me/+e6QTJfwGEus0NThi)
🐳 Kombucha SCOBYs for free (https://t.me/scobysch)

💎 Bienne / Biel (https://t.me/uncommercial_bielbienne)
🏘 Wohnen Biel/Bienne (https://t.me/joinchat/n-50O24IUIxmMWY0)
👥 Mobienne (https://t.me/mobienne)
💎 Thun (https://t.me/joinchat/CaUxwxSJYNG2JxN3wpTJgg)




We could take the description of the groups that is inside the groups and maybe adapt it a bit and make an english text as well and then add it to the website? 😊

I think also just linking the unko list as the source would be good so if it's not up to date anymore people could still find it there?

it's already on the website but it could be mentioned in each group that came from there again so it's more visible?
https://t.me/unkomarkt_sammlung



Also there is some more new things that we did here and maybe some more information that we can use for the website here:

https://chagai.website/notes/visibility

and also maybe here:
https://stayinginbern.ch/event-generator

**Comment by chagai95** [comment]
Let's close this and open a new issue for every groups that we want to add and decide if it's active enough and fitting to the project.
